src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+import { CarouselLists } from '@/constants'
+import Image from 'next/image'
+import { Carousel } from 'react-bootstrap'
+
+export function HomepageCarousel() {
+  return (
+    <div>
+      <Carousel fade>
+        {CarouselLists.map(function (item) {
+          return (
```

Comment:
I thought I request you to use arrow function. Also could we rename the constant name to `CAROUSEL_ITEMS`?
```suggestion
        {CarouselLists.map((item) => (
```

Reply:
done